spec/cc/engine/analyzers/violation_spec.rb

```diff
@@ -0,0 +1,59 @@
+RSpec.describe CC::Engine::Analyzers::Violation, in_tmpdir: true do
+  include AnalyzerSpecHelpers
+
+  describe "#calculate_points" do
+    context "when issue mass exceeds threshold" do
+      it "calculates mass overage points" do
+        language = stub_language(base_points: 100, points_per: 5, mass_threshold: 10)
```

Comment:
What do you think of changing the name to `strategy` and inlining instead of extracting a method?

```ruby
strategy = double(base_points: 100, points_per: 5, mass_threshold: 10)

```